django_declarative_apis/authentication/oauthlib/request_validator.py

```
@@ -61,29 +61,27 @@ def validate_client_key(self, client_key, request):
         self.consumer = oauth_models.get_consumer(client_key)
         if self.consumer is None:
             self.validation_error_message = "consumer_key_unknown"
-            logger.error("invalid consumer")
+            logger.error(
+                'ev=oauth1, error=%s, client_key="%s"',
+                self.validation_error_message,
+                client_key,
+            )
             return False
         return True
 
     def get_client_secret(self, client_key, request):
         try:
             if self.consumer.rsa_public_key_pem:
                 return None
-            else:
-                return self.consumer.secret
-        except Exception as e:  # noqa
-            logger.error(
-                "This should never happen, since consumer is already validated"
```

imo it would make sense to remove the error handling here and let it bubble up, but given this log line has been helpful in the past, can the format of it just be changed?

---

ok I added back the error message, the content slightly updated in [bc69a25](https://github.com/salesforce/django-declarative-apis/pull/168/commits/bc69a25f6e5d9bdabb6e8238ef2db628eca32a83)